src/proj.cpp

```diff
@@ -178,7 +178,7 @@ Rcpp::LogicalVector CPL_is_network_enabled(bool b = false) {
 }
 
 Rcpp::CharacterVector CPL_enable_network(Rcpp::CharacterVector url, bool enable = true) {
-#if PROJ_VERSION_MAJOR >= 7
+#if PROJ_VERSION_MAJOR >= 7 && PROJ_VERSION_MINOR >=1
```

Comment:
That doesn't work for 8.0.x

Reply:
... and we can't run preprocessor on float :thinking:

Reply:
GDAL does [this](https://github.com/r-spatial/sf/blob/master/src/gdal.cpp#L662) kind of things, and it's a nightmare too.